csrc/scheduler/normalization_inner_outer.cpp

```diff
@@ -1128,8 +1135,10 @@ void scheduleReductionCombinedOuter(
         outer_reduction_tv->axis(axisID--)->parallelize(ParallelType::TIDx);
       }
 
-      outer_reduction_tv->split(
-          axisID, NamedScalar::getParallelDim(ParallelType::BIDy));
+      if (rparams.combined_split_grid_inner_dim) {
+        outer_reduction_tv->split(
+            axisID, NamedScalar::getParallelDim(ParallelType::BIDy));
+      }
```

Comment:
ditto

Reply:
same as previous

Reply:
But this one is the same as line 837-841.

(Note, I'm only nitpicking, we don't have to change it. I wanted to point it out in case there's some logic issue here.)

Reply:
Thanks for pointing it out. Yes, they are same and the logic is fine. I repeated them in the if-else branches simplily becuase I want to keep both the code block in if and else branches as complete schedule proess.
**current approach**
```
if(multiple reductions per block){
    schedule approach-1
} else {
    schedule approach-2
}
```
There are some common schedules (e.g. using BIDy) in approach-1 and approach-2. However, it is not prefered to split it out and using the following code:
**Other options:**
```
if(multiple reductions per block){
    part of schedule approach-1
} else {
    part of schedule approach-2
}
common schedule processes shared by approach-1 & approach-2
```